homeassistant/bootstrap.py

```diff
@@ -243,25 +248,26 @@ def async_enable_logging(hass: core.HomeAssistant, verbose: bool=False,
         pass
 
     # Log errors to a file if we have write access to file or config dir
+    global ERROR_LOG_PATH
```

Comment:
Please store data in `hass.data`, a dictionary for this purpose. We should not use globals.